### ISSUE TITLE
Support to arm64 docker image

### ISSUE THREAD

**Issue by hahwul** [noir-cr/noir#271]
*TL;DR*
Noir docker base image: `crystallang/crystal` -> `84codes/crystal`

https://github.com/noir-cr/noir/blob/1fd7b0c64762383198793a275d930bc14d48ab84/Dockerfile#L2

---
Currently, Noir's Docker image does not offer support for aarch64 architecture, primarily due to the lack of aarch64 support in the official Crystal images. It appears that the addition of aarch64 support is not progressing as rapidly as anticipated.

This issue was also a point of consideration in the early development phase of Noir. Given this context, I'm leaning towards the idea of utilizing the community image provided by 84codes[^1] as our base. The [official Crystal documentation](https://crystal-lang.org/install/#docker) mentions 84codes, suggesting it holds a level of trustworthiness akin to official images.

[^1]: https://hub.docker.com/r/84codes/crystal

Should there be any alternative opinions or suggestions, I'm open to discussion!

cc @ksg97031
